pkg/cluster/cluster.go

```diff
@@ -48,11 +49,17 @@ type Config struct {
 	PodServiceAccountRoleBinding *rbacv1beta1.RoleBinding
 }
 
+type ConnectionPoolResources struct {
```

Comment:
maybe `ConnectionPoolChildResources` to not mix it up with cpu/memory resources?

Reply:
Hm, probably event `ConnectionPoolObjects` or something.